doc/changelogs/CHANGELOG_V16.md

```diff
@@ -60,6 +61,336 @@
   * [io.js](CHANGELOG_IOJS.md)
   * [Archive](CHANGELOG_ARCHIVE.md)
 
+<a id="16.16.0"></a>
+
+## 2022-05-XX, Version 16.16.0 'Gallium' (LTS), @TBD
+
+### Notable changes
+
+* **(SEMVER-MINOR)** **crypto**: make authTagLength optional for CC20P1305 (Tobias Nießen) [#42427](https://github.com/nodejs/node/pull/42427)
+* **(SEMVER-MINOR)** **esm**: add chaining to loaders (Jacob Smith) [#42623](https://github.com/nodejs/node/pull/42623)
```

Comment:
This has not landed yet on a Current release line, I don't think it should be there.

Reply:
Yes, please back it out if it’s been included. It has the “backport blocked” label, because we want to release a few breaking-changes PRs together.